downstream/assemblies/hub/assembly-synclists.adoc

```diff
@@ -1,6 +1,15 @@
 [id="assembly-synclists"]
 = Managing Ansible Content Collection synclists in {HubName}
```

Comment:
I wonder if this might be better if changed to "Managing Ansible Content Collections in {HubName}"; then, including the change below?